cmd/ocm/addon-manager/manifests/subscription.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: kuadrant-operator
+  namespace: operators
```

Comment:
is it this ns that is causing the install in the wrong place? what if this was set to kuadrant-system?

Reply:
No i was misunderstanding how operator groups worked this is working now